.github/workflows/release.yml

```diff
@@ -342,6 +342,32 @@ jobs:
         if: always()
         run: find . -type d -not -perm /u+w -exec chmod u+w '{}' \;
 
+  checksums:
+    needs:
+      - release
+      - x64
+      - arm64
+      - armv7
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download assets
+        uses: robinraju/release-downloader@v1.8
+        with:
+          tag: ${{ steps.branch_name.outputs.TAG_NAME }}
+          filename: "*"
+          out-file-path: release_assets
+      - name: Generate checksums
+        run: |
+          sha256sum -b release_assets/* | sed 's/release_assets\///' > checksums.txt
+      - name: Upload checksums
+        uses: shogo82148/actions-upload-release-asset@v1.6.5
+        with:
+          upload_url: ${{ needs.release.outputs.upload_url }}
+          asset_path: checksums.txt
+          asset_name: checksums.txt
```

Comment:
I think the name `sha256sums.txt` would be better. It'd be clearer which hashing algorithm has been used that way.

Reply:
done